.github/workflows/ci.yaml

```diff
@@ -66,6 +66,24 @@ jobs:
           # return to previously saved path
           popd
         done
+    - name: Java Code Coverage
+      timeout-minutes: 20
+      run: |
+        set -x
+        for SERVICE in "balancereader" "ledgerwriter"; do
+          echo "checking $SERVICE..."
+          # save current working dir to memory and cd to src/$SERVICE
+          pushd src/$SERVICE
+            mvn jacoco:report
+            echo "Coverage for $SERVICE:"
+            awk -F, \
```

Comment:
JaCoCo doesn't have a built-in way to show coverage in the console, so I am using a workaround suggested [here](https://stackoverflow.com/questions/49919512/jacoco-coverage-percentage-to-print-on-console)